rcl_logging_spdlog/benchmark/benchmark_logging_interface.cpp

```diff
@@ -0,0 +1,133 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#include <rcutils/allocator.h>
+#include <rcutils/logging.h>
+
+#include <rcl_logging_interface/rcl_logging_interface.h>
+
+#include <string>
+
+#include "performance_test_fixture/performance_test_fixture.hpp"
+
+using performance_test_fixture::PerformanceTest;
+
+namespace
+{
+constexpr const uint64_t kSize = 4096;
+}
+
+const int logger_levels[] =
+{
+  RCUTILS_LOG_SEVERITY_UNSET,
+  RCUTILS_LOG_SEVERITY_DEBUG,
+  RCUTILS_LOG_SEVERITY_INFO,
+  RCUTILS_LOG_SEVERITY_WARN,
+  RCUTILS_LOG_SEVERITY_ERROR,
+  RCUTILS_LOG_SEVERITY_FATAL,
+};
+
+class RCL_LOGGING_BENCHMARK : public PerformanceTest
+{
+public:
+  void SetUp(benchmark::State & st)
+  {
+    performance_test_fixture::PerformanceTest::SetUp(st);
```

Comment:
Unless you think it specifically adds clarity, I think you should drop the namespace here. Either that or add it to the class inheritance so that you're consistent.
```suggestion
    PerformanceTest::SetUp(st);
```